gluoncv/auto/estimators/ssd/ssd.py

```diff
@@ -0,0 +1,282 @@
+"""SSD Estimator."""
+
+import os
+import logging
+import warnings
+import time
+import numpy as np
+import mxnet as mx
+from mxnet import nd
+from mxnet import gluon
+from mxnet import autograd
+import gluoncv as gcv
+
+from gluoncv import data as gdata
+from gluoncv import utils as gutils
+from gluoncv.model_zoo import get_model
+from gluoncv.data.batchify import Tuple, Stack, Pad
+from gluoncv.data.transforms import presets
+from gluoncv.data.transforms.presets.ssd import SSDDefaultTrainTransform
+from gluoncv.data.transforms.presets.ssd import SSDDefaultValTransform
+from gluoncv.data.transforms.presets.ssd import SSDDALIPipeline
+from gluoncv.nn.bbox import BBoxClipToImage
+from gluoncv.utils.metrics.voc_detection import VOC07MApMetric
+from gluoncv.utils.metrics.coco_detection import COCODetectionMetric
+from gluoncv.utils.metrics.accuracy import Accuracy
```

Comment:
should use relative imports as estimators/ is under the gluoncv folder.  Eg. from ...model_zoo import get_model

Reply:
we should use relative import here